docs/spec.md

```diff
@@ -1302,8 +1302,9 @@ y []any
 
 ## Type Assertion
 
-A type assertion `ident.(type)` asserts that the value of the variable
-`ident` is of the given `type`. If the assertion does not hold a
+A type assertion `x.(TYPE)` asserts that the value of the variable `x`
+is of the given `TYPE`. `TYPE` can be any basic or composite type, such
+as `num` or `[]string`. If the assertion does not hold a
```

Comment:
I think I would put a comma after "hold" to disambiguate "does not hold" and "does not hold a <noun>"

Reply:
agreed, done.